java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/CPMapTest.java

```diff
@@ -35,16 +45,26 @@ public class CPMapTest extends HazelcastTest {
     public int maps = 1;
     // number of distinct keys to create and use per-map; key domain is [0, keys)
     public int keys = 1;
+    // number of distinct value to create on every client (total number of values is valuesPerClient * clients)
+    public int valuesPerClient = 1;
     // size in bytes for each key's associated value
     public int valueSizeBytes = 100;
+    // set it to true if test should check whether all values have been changed during the test run.
+    public boolean checkInitialValueChanged = false;
+
+    private static final String INITIAL_VALUE = "initialValue";
```

Comment:
Just pick a value from the 'v' array so that the value stored, always is consistent with the configuration.

If we set value size to 1, then on startup the length of the values will be lot larger than 1.

Reply:
Yes, but that's the value which is expected to be changed during test running. 

What we can do is to change it the logic based on value of `checkInitialValueChanged` - if `checkInitialValueChanged=false` then we don't care about initial value and we can set here anything from `v[]`. If we care about initial value (`checkInitialValueChanged=true`) we can set here this constant. WDYT?

Reply:
I don't like it. It just adds more complexity.

Reply:
Is it really to complex? This is just one option which basically saying if it's true I want to put initial data to the map. Moreover it will be disabled by default, so if you don't care about it you don't need to do anything. Sounds quite straight-forward for me.